packages/headless/src/controllers/core/quickview/headless-core-quickview.ts

```diff
@@ -65,18 +65,18 @@ export interface QuickviewCoreState {
 }
 
 /**
- * Creates a Quickview controller core instance.
+ * Creates a `Quickview` controller instance.
  *
  * @param engine - The headless engine.
- * @param props - The configurable `QuickviewCore` properties.
+ * @param props - The configurable `Quickview` properties.
  * @param fetchResultContentCallback - The callback to be triggered after executing fetchResultContent.
- * @returns A `QuickviewCore` controller instance.
+ * @returns A `Quickview` controller instance.
  */
-export function buildQuickviewCore(
+export function buildCoreQuickview(
```

Comment:
👍 for bringing it back to the standard name :) CoreQuickview instead of QuickviewCore